deeppavlov/configs/sentiment/insults_kaggle.json

```diff
@@ -0,0 +1,135 @@
+      "x"
+    ],
+    "in_y": [
+      "y"
+    ],
+    "pipe": [
+      {
+        "id": "classes_vocab",
+        "name": "default_vocab",
+        "fit_on": [
+          "y"
+        ],
+        "level": "token",
+        "save_path": "vocabs/insults_kaggle_classes.dict",
+        "load_path": "vocabs/insults_kaggle_classes.dict"
+      },
+      {
+        "in": [
+          "x"
+        ],
+        "out": [
+          "x_prep"
+        ],
+        "name": "dirty_comments_preprocessor"
+      },
+      {
+        "id": "my_embedder",
+        "name": "fasttext",
+        "save_path": "embeddings/wordpunct_tok_reddit_comments_2017_11_300.bin",
+        "load_path": "embeddings/wordpunct_tok_reddit_comments_2017_11_300.bin",
+        "dim": 300
+      },
+      {
+        "id": "my_tokenizer",
+        "name": "nltk_tokenizer",
+        "tokenizer": "wordpunct_tokenize"
+      },
+      {
+        "in": [
+          "x_prep"
+        ],
+        "in_y": [
+          "y"
+        ],
+        "out": [
+          "y_labels",
+          "y_probas_dict"
+        ],
+        "main": true,
+        "name": "intent_model",
+        "save_path": "sentiment/insults_kaggle_v0",
+        "load_path": "sentiment/insults_kaggle_v0",
+        "classes": "#classes_vocab.keys()",
+        "kernel_sizes_cnn": [
+          1,
+          2,
+          3
+        ],
+        "filters_cnn": 256,
+        "confident_threshold": 0.5,
+        "optimizer": "Adam",
+        "lear_rate": 0.01,
+        "lear_rate_decay": 0.1,
+        "loss": "binary_crossentropy",
+        "last_layer_activation": "softmax",
+        "text_size": 100,
+        "coef_reg_cnn": 1e-3,
+        "coef_reg_den": 1e-2,
+        "dropout_rate": 0.5,
+        "dense_size": 100,
+        "model_name": "cnn_model",
+        "embedder": "#my_embedder",
+        "tokenizer": "#my_tokenizer"
+      }
+    ],
+    "out": [
+      "y_labels",
+      "y_probas_dict"
+    ]
+  },
+  "train": {
+    "epochs": 1000,
+    "batch_size": 64,
+    "metrics": [
+      "classification_accuracy",
+      "classification_f1",
+      "classification_roc_auc"
+    ],
+    "validation_patience": 5,
+    "val_every_n_epochs": 5,
+    "log_every_n_epochs": 5,
+    "show_examples": false,
+    "validate_best": true,
+    "test_best": true
+  },
+  "metadata": {
+    "labels": {
+      "telegram_utils": "IntentModel",
+      "server_utils": "KerasIntentModel"
+    },
+    "download": [
+      "http://lnsigo.mipt.ru/export/deeppavlov_data/vocabs.tar.gz",
+      "http://lnsigo.mipt.ru/export/deeppavlov_data/sentiment.tar.gz",
+      {
+        "url": "http://lnsigo.mipt.ru/export/datasets/insults_data/train.csv",
+        "subdir": "insults_data"
+      },
+      {
+        "url": "http://lnsigo.mipt.ru/export/datasets/insults_data/valid.csv",
+        "subdir": "insults_data"
+      },
+      {
+        "url": "http://lnsigo.mipt.ru/export/datasets/insults_data/test.csv",
+        "subdir": "insults_data"
+      },
```

Comment:
You can add `train.csv`, `valid.csv` and `test.csv` into a single `tar.gz` archive